docs/guides/configure-mcp-gateway-listener-and-router.md

```
@@ -5,33 +5,32 @@ This guide covers adding an MCP listener to your existing Gateway. The controlle
     port: 8080
     protocol: HTTP
     allowedRoutes:
       namespaces:
         from: All
+'
 ```
 
+Replace `your-gateway-name`, `your-gateway-namespace`, and the hostname with your values. The hostname must resolve to your Gateway's external address.
+
+> **Note:** The patch above replaces all listeners. To preserve existing listeners, use a JSON patch or edit the Gateway directly with `kubectl edit gateway your-gateway-name -n your-gateway-namespace`.
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)